src/main/java/game/RacingGame.java

```diff
@@ -19,16 +21,18 @@ public RacingGame(MovePolicy movePolicy, RacingGameInputView racingGameInputView
     }
 
     public void play() {
-        int numberOfCar = racingGameInputView.getNumberOfCar();
+        List<CarName> carNameList = racingGameInputView.getCarNameList();
         int time = racingGameInputView.getTime();
         List<Car> carList = new ArrayList<>();
-        IntStream.range(0, numberOfCar).forEach(i -> carList.add(new Car(i, 0, movePolicy)));
+        carNameList.forEach(carName -> carList.add(new Car(carName, INIT_POSITION, movePolicy)));
 
         for (int i = 0; i < time; i++) {
-            racingGameResultView.printRound(i+1);
+            racingGameResultView.printRound(i + 1);
             carList.forEach(Car::move);
-            carList.forEach(car -> racingGameResultView.printResult(car.getPosition()));
+            carList.forEach(racingGameResultView::printResult);
         }
 
+        int winScore = carList.stream().max((o1, o2) -> o1.getPosition() - o2.getPosition()).map(Car::getPosition).orElse(INIT_POSITION);
```

Comment:
다음과 같이 바꿀 수 있을 것 같네요 :)
```suggestion
        int winScore = carList.stream()
                              .max(Comparator.comparingInt(Car::getPosition))
                              .map(Car::getPosition)
                              .orElse(INIT_POSITION);
```

Reply:
이것도 5단계 PR 에서 반영하였습니다 ~